source/behaviors/autosave/autosave.tests.ts

```diff
@@ -192,14 +192,5 @@ describe('AutosaveDirective', () => {
 			sinon.assert.calledWith(autosaveAction.trigger, waitValue);
 		});
 
-		it('should not save if the form becomes pristine immediately before saving', () => {
```

Comment:
We still want this. This was another thing that got corrected/added with the previous PR